mentors/python/p.json

```
@@ -23,4 +23,12 @@
     "avatar_url": null,
     "bio": "I am a fullstack software engineer specialising in python and javascript related technologies and a huge enthusiast of the deep learning revolution. My aim is to help people, as I would like to have been offered help when I was doing my first steps in the field."
   }
```

```suggestion
  },
```

Just missing a comma to separate your bio from the previous person's bio.

---

Alright, sure thing